fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBRecordStoreBase.java

```diff
@@ -959,6 +999,20 @@ default RecordCursor<FDBIndexedRecord<M>> scanIndexRecordsEqual(@Nonnull final S
         return scanIndexRecords(indexName, IndexScanType.BY_VALUE, range, null, ScanProperties.FORWARD_SCAN);
     }
 
+    /**
+     * Scan the records pointed to by an index equal to indexed values using the Index Prefetch method.
+     * @param indexName the name of the index
+     * @param values a left-subset of values of indexed fields
+     * @return a cursor of the records pointed to by the index
+     */
+    @Nonnull
+    default RecordCursor<FDBIndexedRecord<M>> scanIndexRemoteFetchRecordsEqual(@Nonnull final String indexName, KeyExpression primaryKey, @Nonnull final Object... values) {
```

Comment:
I think this one missed the `EXPERIMENTAL` label when the others got updated

Reply:
Done.